app/models/patient/triage_status.rb

```diff
@@ -81,6 +81,10 @@ def vaccination_history_requires_triage?
     end && !vaccinated?
   end
 
+  def specifies_delivery_method?
+    safe_to_vaccinate? && vaccine_method.present?
+  end
```

Comment:
I find the naming of this method a little confusing, but I'm also wondering if everywhere that we do `specifies_delivery_method?`, we can do `vaccine_method.present?` instead because they'll only be present if the patient is safe to vaccinate.